app/router.js

```diff
@@ -8,4 +8,5 @@ export default class Router extends EmberRouter {
 
 Router.map(function() {
   this.route('signup');
+  this.route('notification')
```

Comment:
Can we have `notifications` (plural)?

Reply:
@swarajpure @adnan1308 Can you please tell me why you used  singular?